alerts/cluster-image-registry-operator/OWNERS

```diff
@@ -0,0 +1,5 @@
+reviewers:
+- flavianmissi
+approvers:
+- flavianmissi
```

Comment:
you probably want at least another approver/reviewer.

Reply:
Yes I'd certainly like that. Unfortunately there is no one else working in the image registry, so there is no one else to add to either list :(

Reply:
Ah bummer! Nobody from docs, PM, QE who could help with reviews?

Reply:
Good question - @xiuwang would you be up for being added to the OWNERS here? 🌟

Reply:
from personal experience, QE and docs are very helpful reviewers because they can pinpoint issues which are less visible for engineers.

Reply:
Sure, @wewang58 and me both could be added to the OWNERS.

Reply:
added our docs person (@\stevsmit) to OWNERS as well. @simonpasquier PTAL

Reply:
If you split the OWNERS file in another PR, you can get it merged fast and then iterate on the content of the runbooks at your own pace.